ui/components/app/modals/visit-support-data-consent-modal/visit-support-data-consent-modal.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import VisitSupportDataConsentModal from '.';
+
+export default {
+  title: 'Components/App/Modals/VisitSupportDataConsentModal',
+};
+
+export const DefaultStory = () => (
+  <VisitSupportDataConsentModal
+    version="1.0.0"
```

Comment:
this line is not necessary because we're reading from the env var inside the component

Reply:
Good catch ! 🕵🏻‍♂️ 915721b0ce087a8c5afa8f76ca5d04db4114e233